packages/vite-plugin/src/node/plugin-manifest.ts

```diff
@@ -209,7 +209,11 @@ export const pluginManifest: CrxPluginFn = () => {
         } else {
           // vite build emits content scripts, html files and service worker
           if (manifest.content_scripts)
-            for (const { js = [], matches = [] } of manifest.content_scripts)
+            for (const {
+              js = [],
+              matches = [],
+              world = 'ISOLATED',
```

Comment:
dont we need to pass the world argument into the first if leg? `if (config.command === 'serve') {`